### PR TITLE
Replace missing field names from status Metricset in MongoDB Dashboard

### PR DESCRIPTION
**Pull request by sayden**
According to issue https://github.com/elastic/beats/issues/9715 it seems that some field names within the MongoDB dashboard were incorrect.

It seems that this commit changed the field names https://github.com/elastic/beats/commit/5544838e2002f6a8bf956bafc25d0df8f34f5437 5 months ago.

In this PR, the _correct_ field names are set. I'm not very experienced with Kibana so I have just modified current dashboard JSON file instead of replacing it with the exported one using [this method](https://www.elastic.co/guide/en/beats/devguide/current/export-dashboards.html).

Working in Kibana 6.5.4

### CODE REVIEW

File: metricbeat/module/mongodb/_meta/kibana/6/dashboard/Metricbeat-mongodb-overview.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
                             "enabled": true, 
                             "id": "2", 
                             "params": {
-                                "field": "metricset.host", 
+                                "field": "host.name",
```
Comment (ruflin) [inline metricbeat/module/mongodb/_meta/kibana/6/dashboard/Metricbeat-mongodb-overview.json]: `metricset.host` and `host.name` are not the same thing. `host.name` is then name from the host on which the beat runs, `metricset.host` is what address of the service it connects to.

These fields should not have changed in the PR your linked. Same below.

Comment (sayden) [reply metricbeat/module/mongodb/_meta/kibana/6/dashboard/Metricbeat-mongodb-overview.json]: Ok, I have reverted both ocurrences of `host.name` to their original values `metricset.host`

**Comment by sayden** [comment]
Yes, of course. That's how I realized that `host.name` was missing in the dashboard. So maybe there's something else that is wrong?

I'll wait for your answer before merging.

**Comment by ruflin** [comment]
@sayden All good. Please merge and backport.

**Pull request merged**